etc/databases/world/updates/updates.sql

```diff
@@ -30682,5 +30682,27 @@ begin not atomic
 
         insert into applied_updates values ('110320231');
     end if;
+
+    -- 12/03/2023 1
+    if(select count(*) from applied_updates where id = '120320231') = 0 then
+
+        -- Set Coldridge Mountaineer's movement type to idle instead of wandering.
+        update spawns_creatures set movement_type = 0 where spawn_id = 398;
+
+        -- Despawn Felix' items as the quest doesn't yet exist.
+        update spawns_gameobjects set ignored = 1 where spawn_id in(1394, 1386, 1380);
+
+        -- Despawn unused Mortar object (Shane Cube).
+        update spawns_gameobject set ignored = 1 where spawn_id = 1175;
+
+        -- Set correct faction for Magis Stonemantle <Mage Trainer>.
+        -- He's a dwarf, not a gnome.
+        update creature_template set faction_id = 57 where entry = 1228;
```

Comment:
In Vanilla he actually uses faction 875 which is for gnomes. But probably he should use dwarf faction in 0.5.3 unless we got the old one from sniffs. Let me verify.